daisy_workflows/image_build/enterprise_linux/kickstart/el7-post.cfg

```diff
@@ -31,20 +40,16 @@ EOL
 # Disable IPv6 for Yum.
 echo "ip_resolve=4" >> /etc/yum.conf
 
-# Add GPG keys for GCE repos.
-curl -o /etc/pki/rpm-gpg/google-rpm-package-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
-curl -o /etc/pki/rpm-gpg/google-key.gpg https://packages.cloud.google.com/yum/doc/yum-key.gpg
-rpmkeys --import /etc/pki/rpm-gpg/google-rpm-package-key.gpg
-rpmkeys --import /etc/pki/rpm-gpg/google-key.gpg
-
 # Install GCE guest packages and CloudSDK.
-yum install -y google-compute-engine google-osconfig-agent google-cloud-sdk gce-disk-expand
+yum install -y google-compute-engine google-osconfig-agent gce-disk-expand
+yum install -y google-cloud-sdk
 
 # Send /root/anaconda-ks.cfg to our logs.
 cp /run/install/ks.cfg /tmp/anaconda-ks.cfg
 
-# Remove files which shouldn't make it into the image.
-rm -f /etc/boto.cfg /etc/udev/rules.d/70-persistent-net.rules
+# Remove files which shouldn't make it into the image. These files may not
+# exist.
+rm -f /etc/boto.cfg /etc/udev/rules.d/70-persistent-net.rules || return 0
```

Comment:
`rm -f` returns 0 on missing files. if that's the only fail scenario to handle, you can omit `|| return 0`
also do we want to *return* if one fails? i would think we'd want to continue?

Reply:
Yeah- your'e right. Changed.